config/staging.yaml

```diff
@@ -31,4 +31,10 @@ node-groups:
 bee-configs:
   staging:
     _inherit: "default"
-    bootnodes: "/dnsaddr/mainnet.ethswarm.org"
+    swap-endpoint: http://rpc-sepolia-haproxy.default.svc.swarm1.local
```

Comment:
@darkobas2 can we use this endpoint in case of setting up "our" staging env that will be connected to sepolia?

Reply:
This will work on our kubernetes clusters... Not on a local cluster or so.

Reply:
This setup is for our kubernetes clusters, for example to deploy it in some of our namespaces..

Reply:
Yes